src/execute.cpp

```diff
@@ -108,13 +114,16 @@ win32execute(const char* path,
   std::string args = Win32Util::argv_to_string(argv, sh);
   std::string full_path = Win32Util::add_exe_suffix(path);
   std::string tmp_file_path;
+  const char* lpAppName = full_path.c_str();
```

Comment:
I suggest continuing to use `full_path.c_str()` instead of introducing an alias. If another variable is still wanted for some reason, please follow the naming convention (i.e. name it something like `full_path_str`).

Also, perhaps the compiler path should be double quoted, both in the first and second argument to `CreateProcess`?

Reply:
I'll remove this alias, sure. I was doing a lot of LOG() debugging and probably I added this variable for debugging.
As for quoting the path - I don't want to add it in this pull request. It works for me as it is. I can analyze it later (just to divide work into smaller chunks).